chandra_aca/star_probs.py

```diff
@@ -704,17 +717,16 @@ def t_ccd_warm_limit_for_guide(mags, min_guide_count=4.0, warm_t_ccd=-5.0, cold_
     """
     if guide_count(mags, warm_t_ccd) >= min_guide_count:
         return warm_t_ccd
-    if guide_count(mags, cold_t_ccd) < min_guide_count:
-        # Note that this uses a '<' not a '<=' because if the guide_count is equal to
-        # min_guide_count at cold_t_ccd, we still want to solve for a warmer temperature
-        # when it is != to min_guide_count
+    if guide_count(mags, cold_t_ccd) <= min_guide_count:
+        # Note that this relies on a slight incline in the guide_count curve
+        # from 1.0005 at mag=6.0 to 1.0 at mag=10.0.
         return cold_t_ccd
 
     def merit_func(t_ccd):
         count = guide_count(mags, t_ccd)
-        # A small number (0.01) is added to (count - min_guide_count) so that the function
+        # A small number d_count is added to (count - min_guide_count) so that the function
```

Comment:
Comment lies! :smile:

Reply:
I can juts cut the comment altogether if there's no longer a need for any hack to get distinct zero crossings.